site/content/docs/commands/env-init.en.md

```diff
@@ -29,26 +29,44 @@ Import Existing Resources Flags
       --import-vpc-id string             Optional. Use an existing VPC ID.
 
 Configure Default Resources Flags
-      --override-private-cidrs strings   Optional. CIDR to use for private subnets (default 10.0.2.0/24,10.0.3.0/24).
-      --override-public-cidrs strings    Optional. CIDR to use for public subnets (default 10.0.0.0/24,10.0.1.0/24).
-      --override-vpc-cidr ipNet          Optional. Global CIDR to use for VPC (default 10.0.0.0/16).
+      --override-az-names strings        Optional. Availability Zone names.
+                                         (default 2 random AZs)
+      --override-private-cidrs strings   Optional. CIDR to use for private subnets.
+                                         (default 10.0.2.0/24,10.0.3.0/24)
+      --override-public-cidrs strings    Optional. CIDR to use for public subnets. 
+                                         (default 10.0.0.0/24,10.0.1.0/24)
+      --override-vpc-cidr ipNet          Optional. Global CIDR to use for VPC.
+                                         (default 10.0.0.0/16)
 
-Global Flags
-  -a, --app string   Name of the application.
+Telemetry Flags
+      --container-insights   Optional. Enable CloudWatch Container Insights.
 ```
 
 ## Examples
-Creates a test environment in your "default" AWS profile using default config.
+Creates a test environment using your "default" AWS profile and default configuration.
 ```bash
 $ copilot env init --name test --profile default --default-config
 ```
 
-Creates a prod-iad environment using your "prod-admin" AWS profile using existing VPC.
+Creates a prod-iad environment using your "prod-admin" AWS profile and enables container insights.
```

Comment:
We should probably capitalize "Container Insights" in the help menu, maybe even "CloudWatch Container Insights"?